R/calculate_tmb.R

```diff
@@ -31,7 +31,7 @@
 #'     maf1,
 #'     regions_bed = grch37_ashm_regions
 #' )
-#' #' calculate_tmb(
+#' calculate_tmb(
```

Comment:
The title of this function should probably be "calculate tumour mutation burden", to be more intuitive and self-explanatory

Reply:
Thanks - Updated the title!